test_notebooks/trajectory_visualization.py

```diff
@@ -198,12 +179,14 @@ def shift_trajectory(xydata):
 
                 justright = False
 
```

Comment:
Is there a way to figure out which columns have what content in these numpy arrays?  Even if they're not labeled, they could probably be recognized due to their erratic numbers, and they'll probably be adjacent with x first and y second.  Alternatively, the user must have some knowledge of which column contains what: you might put the onus on the user to input to the function which columns contain the x and y data.  That would be more of a pain but would enable more flexibility for later or differently formatted datasets.